app/models/user.rb

```diff
@@ -1,8 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly enable `:confirmable` for the devise, this will allow for user to confirm their email on sign up

Reply:
- [ ] As the last reviewer said you have to enable `:confirmable` on the device so that the users can confirm their emails.